app/boards/frdm_mcxn947_mcxn947_cpu0.overlay

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+        cannectivity: cannectivity {
+		compatible = "cannectivity";
+		timestamp-counter = <&ctimer0>;
+
+		channel0 {
+			compatible = "cannectivity-channel";
+			can-controller = <&flexcan_fd0>;
+			termination-gpios = <&gpio0 10 GPIO_ACTIVE_LOW>; /* Red LED for testing */
+			state-led = <&blue_led>;
+			activity-leds = <&green_led>;
+		};
+	};
+
+        gs_usb0: gs_usb0 {
+                compatible = "gs_usb";
+                label = "gs_usb";
+        };
+
+};
```

Comment:
Apart from the testing LEDs, this is the default configuration. With the CAN FD customisation out of the way, no overlay should be needed for this board.

Reply:
the default can dts is not for canFD on this board, I need customize for this.

Reply:
The CAN FD support needs to go into Zephyr main for this SoC (and the tests need to pass for this).

Enabling CAN FD here gives a false idea that CAN FD is working as it should on this SoC (with the current HAL and driver support) and will just lead to error reports for CANnectivity down the road.

Reply:
thanks, I will wait till this issue is resolved

Reply:
Ok. Could you please split the compilation fix out in a separate PR then?